dts/arm/atmel/sam3x-pinctrl.dtsi

```diff
@@ -10,6 +10,16 @@
 	soc {
 		pinctrl@400e0e00 {
 			/* instance, signal, pio, pin, peripheral */
+			DT_ATMEL_PIN(spi0, miso, a, 25, a);
+			DT_ATMEL_PIN(spi0, mosi, a, 26, a);
+			DT_ATMEL_PIN(spi0, npcs0, a, 28, a);
+			DT_ATMEL_PIN(spi0, npcs1, a, 29, a);
+			DT_ATMEL_PIN(spi0, npcs1, b, 20, b);
+			DT_ATMEL_PIN(spi0, npcs2, a, 30, a);
+			DT_ATMEL_PIN(spi0, npcs2, b, 21, b);
+			DT_ATMEL_PIN(spi0, npcs3, a, 31, a);
+			DT_ATMEL_PIN(spi0, npcs3, b, 23, b);
+			DT_ATMEL_PIN(spi0, spck, a, 27, a);
```

Comment:
Agreed! Since SAM3X8H device is not commercially available we don't need add spi1.
I noted that there isn't sam3x.dtsi updates. I presume that never worked on this board.
Just for you know, Atmel SPI is the only component that probably will work between all SAM variations (not SAM0) without effort. We can add it on future.

Reply:
I removed spi1 already.